cmake/cmake/External_eigen.cmake

```diff
@@ -0,0 +1,10 @@
+ExternalProject_Add(eigen
+    GIT_REPOSITORY  https://gitlab.com/libeigen/eigen.git
```

Comment:
This could point to the local eigen/fmt/Expected submodules to not break anything.

Reply:
I'll try this in a follow-up!